record.py

```diff
@@ -125,7 +125,7 @@ def draw(self):
         # Image
         img = self.bmp.ConvertToImage()
         img = img.Rescale(320,240)
-        self.image_widget.SetBitmap( img.ConvertToBitmap() )
+        self.image_widget.SetBitmap(self.bmp)
```

Comment:
I kind of want to rename this to `self.screenshot` now that I return to this code. What do you think?